deploy/kubernetes/fluentd-sumologic.yaml.tmpl

```diff
@@ -656,3 +656,192 @@ spec:
           value: "100k"
         - name: TOTAL_LIMIT_SIZE
           value: "128m"
+---
+# Source: sumologic/templates/prometheus-rules.yaml
```

Comment:
Huh, I think you'll need to open a new PR :( CI doesn't seem smart enough to remove this

Reply:
Boo.  will do